data-transform.qmd

```diff
@@ -335,6 +350,8 @@ In this situation, the first challenge is often just focusing on the variables y
       select(!year:day)
     ```
 
+    You can use `-` instead of `!` in the code above, but we recommend `!` to indicate "not" in a `select()` statement.
```

Comment:
```suggestion
    You can also use `-` instead of `!` (and you're likely to see that in the wild); we recommend `!` because it reads as "not", and combines well with `&` and `|`.
```